ui/tests/integration/components/sync/sync-header-test.js

```diff
@@ -32,30 +32,42 @@ module('Integration | Component | sync | SyncHeader', function (hooks) {
     assert.dom(breadcrumb).includesText('Secrets Sync', 'renders default breadcrumb');
   });
 
-  test('it should render breadcrumbs', async function (assert) {
+  test('it should render passed in breadcrumbs', async function (assert) {
     this.breadcrumbs = [{ label: 'Destinations', route: 'destinations' }];
     await this.renderComponent();
     assert.dom(breadcrumb).includesText('Destinations', 'renders breadcrumb');
   });
 
-  test('it should just render title for enterprise with secrets sync feature', async function (assert) {
-    this.version.type = 'enterprise';
-    this.version.features = ['Secrets Sync'];
-    await this.renderComponent();
-    assert.dom(title).hasText('Secrets Sync');
-  });
+  module('enterprise', function (hooks) {
```

Comment:
Again, this will not run on our OSS PRs, even though it's a component test so it could. The setup in `beforeEach` has everything you should need to run this in OSS instead so this module could be renamed to `ent`